### ISSUE TITLE
Abundance profile format contradicts given example

### ISSUE THREAD

**Issue by jvollme** [younglululu/COCACOLA#5]
Your script states that abundance profiles should be in a format where each line corresponds to a contig and each column corresponds to a sample. 
As an example you  show how to call CONCOCTS `gen_input_table.py` to create a coverage-profile in tabular format. 

However, your in your example, this script is simply run without the subsequent parsing step which is required for CONCOCT. Therefore the profile-table would include an extra column at the beginning of the table. containing each contig's length. This would contradict your format specification above. In the CONCOCT example workflow, this "length" column is removed from the table using `cut -f1-3-` and the result is piped into the final input file "input-tableR.tsv". 

Based on the naming scheme used in your example and on the format specification above, I would assume that the standard result of CONCOCTS  `gen_input_table.py` (without subseqeunt processing/parsing) would NOT be correct input for COCACOLA?

**Comment by younglululu** [comment]
You are right, COCACOLA indeed assumes that the extra column containing each contig's length is excluded, sorry for the confusion. 

To solve this, in python version of COCACOLA, I would suggest to change line 157 of cocacola.py from

covMat = pd.read_csv(args.abundance_profiles,sep='\t',usecols=range(1, covHeader.shape[1])).as_matrix()

to 

covMat = pd.read_csv(args.abundance_profiles,sep='\t',usecols=range(2, covHeader.shape[1])).as_matrix()

The meaning is to start from the third column (of index 2) instead of the second column (of index 1).
